docker-dynamodb/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+
+services:
+  db:
+    image: constellation-db
+    deploy:
+      replicas: 1
+      resources:
+        cpus: 0.1
+        memory: 100M
+    volumes:
+      - ./data:/opt/dynamodb/data
```

Comment:
Just to double check my understanding here - this is a file on the host machine that we're allowing the container to access?

So sharing dbs would amount to devs giving each other copies of this file?